sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -56,9 +76,9 @@ az cognitiveservices account keys list --resource-group <your-resource-group-nam
 Once you have an API key and endpoint, you can use it as follows:
 
 ```js
-const { CustomFormClient, ApiKeyCredential } = require("@azure/ai-form-recognizer");
```

Comment:
ApiKeyCredential -> AzureKeyCredential

Reply:
This one is already AzureKeyCredential, but I did find another one reference in the listing models sample